### ISSUE TITLE
Do not do "External perimeter first" in the first (several) layers

### ISSUE THREAD

**Issue by zhou13** [supermerill/Slic3r#4375]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated with perimeter separation in the first several layers in ABS printing when using External perimeter first printing.

Left side: printed with "External perimeter first". I got Perimeter separation consistently.
Right side: printed without "External perimeter first". Perimeter separation problems go away.
![IMG_2033](https://github.com/user-attachments/assets/ed19ca0e-28b6-4bad-8b66-8da99ade183e)

**Describe the solution you'd like**
Add an option to disable "External perimeter first" in the first (several) layers.

**Describe how it would work**
Since we are not doing "External perimeter first" in the first several layers, the perimeter separation problem should go away.

**Describe alternatives you've considered**
I can adjust "better bonding" but it might cause other issues. This is also related to first layer xy compensation but if I disable that I might suffer from the elephant foot issue.

**Comment by ghost** [comment]
I'll add a vote for this, I've got a few models that have this problem, and its kind of annoying to have to pick off or on.  For example the ERCF v2, buffer support arm really struggles with external perimeters first on the first few layers, but otherwise doesnt really even have overhangs, and prints fine after it figures things out :)

**Comment by legend069** [comment]
does that model have a chamfer/fillet ?
going to guess that's whats causing the issue more then it being on for the first layer/s

**Comment by zhou13** [comment]
Yes, it is the chamfer plus the potential my 0.1mm first layer xy compensation that is causing the issue.

**Comment by ghost** [comment]
same, dont have an issue doing overhangs and such after the first 2-3 layers though, so I see this as a useful configuration option in some cases.  Usually it doesnt cause issues, but certain geometries the filament tension seems to pull it off the thin overlap with the previous layers thanks the the chamfer and angles involved.

**Comment by zhou13** [comment]
![image](https://github.com/user-attachments/assets/c867ff5d-c1a3-4a34-8057-3c8d13b5736d)

Add more photos that the option causes separation on the first layer.
